modules/git.py

```diff
@@ -204,20 +205,23 @@ def do_POST_unsafe(self, data):
             if config.git_events:
                 full_name = data['repository']['full_name']
                 event_types = []
-
                 for key, value in config.git_events.items():
                     if fnmatch(full_name, key):
                         event_types = value
             else:
                 event_types = None
 
-            if (event_types is not None) and (event not in event_types):
+            event_specification_exists = False
```

Comment:
```suggestion
            event_in_config = False
```

or something like that. Let's not use both the word spec and config.